test-cases/javascript/interactive/multi-step-request-event-attribute.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body>
+    <title>CrawlMaze - Testbed for Web Crawlers - multi-step request with event attribute </title>
+    <h1> multi-step GET method </h1>
+    <input type="button" onClick="showSecretButton();" value='show secret button'/>
```

Comment:
I've just noticed that sometimes you use `"` and sometimes `'`. You can replace all with double quotes to stay consistent